client/src/app/core/ui-services/tree.service.ts

```diff
@@ -46,6 +46,8 @@ export interface FlatNode {
     isSeen: boolean;
     expandable: boolean;
     id: number;
+    tag?: any;
```

Comment:
What is tag used for?

Reply:
`tag` is used for something a component can search for or to filter the node(s).